Wikipedia/Code/FindInPageKeyboardBar.h

```diff
@@ -0,0 +1,24 @@
+#import <UIKit/UIKit.h>
+
+@class FindInPageKeyboardBar;
+
+@protocol FindInPageBarDelegate <NSObject>
+
+- (void)findInPageTermChanged:(NSString *)text sender:(FindInPageKeyboardBar *)sender;
```

Comment:
"Sender" is a terminology that is usually used with target/action or responder chain. With the delegate pattern, the name of the class is preferred, here I would use something like "keyboardBar".

Also with delegate methods, it is convention to always pass the object back as the the first parameter. See the [UITableViewDelegate](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UITableViewDelegate_Protocol/index.html#//apple_ref/occ/intf/UITableViewDelegate) or the [UITextViewDelegate](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UITextViewDelegate_Protocol/index.html#//apple_ref/doc/uid/TP40006897) protocols for an example pattern.